app/assets/javascripts/miq_debug.js

```diff
@@ -1,24 +1,16 @@
 // CTRL+SHIFT+X stops the spinner
 $(document).bind('keyup', 'ctrl+shift+x', miqSparkleOff);
 
-// CTRL+SHIFT+Z checks for duplicate element ids
-$(document).bind('keyup', 'ctrl+shift+z', function() {
-  var ids = {};
+/// Warn for duplicate DOM IDs
+(function () {
+  var duplicate = function () {
+    $('[id]').each(function(){
+      var ids = $('[id="' + this.id + '"]');
+      if(ids.length > 1 && ids[0] == this)
```

Comment:
`ids[0] == this` .. Are we sure that it will always be first here? Maybe better to use `ids.indexOf(this) !== -1`?

And space after `if` please ;)